configs/terraform/environments/prod/gitleaks.tf

```diff
@@ -6,33 +6,24 @@ resource "google_service_account" "gitleaks_secret_accesor" {
   description  = "Identity of gitleaks. It's used to retrieve secrets from secret manager"
 }
 
-data "github_repository" "gitleaks_repository" {
-  for_each = var.gitleaks_repositories
-  provider = github.kyma_project
-  name     = each.value
-}
-
 resource "google_service_account_iam_binding" "gitleaks_workload_identity_federation_binding" {
-  for_each = data.github_repository.gitleaks_repository
   members = [
-    "principal://iam.googleapis.com/projects/${data.google_client_config.gcp.project}/locations/global/workloadIdentityPools/${module.gh_com_kyma_project_workload_identity_federation.pool_name}/subject/repository_id:${each.value.repo_id}:repository_owner_id:${var.github_kyma_project_organization_id}:workflow:${var.gitleaks_workflow_name}"
+    "principal://iam.googleapis.com/projects/${data.google_client_config.gcp.project}/locations/global/workloadIdentityPools/${module.gh_com_kyma_project_workload_identity_federation.pool_name}/subject/repository_owner_id:${var.github_kyma_project_organization_id}:workflow:${var.gitleaks_workflow_name}"
```

Comment:
Can we use data for kyma-project organization id instead using variable?

Reply:
The data source for github organizaiton doesn't provide such information. We have to stick with variable until we found different provider